### PR TITLE
fix: Fix CI workflows

### PR DESCRIPTION
**Pull request by joaofnds**
gif-or-image

### What?
These are the things we'd had to change to make the workflow run on [skore-io/lives](https://github.com/skore-io/lives). These changes were added on [skore-io/lives#5](https://github.com/skore-io/lives/pull/5)

### CODE REVIEW

File: .k8s/deployment_web.tpl.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ spec:
           livenessProbe:
             failureThreshold: 3
             httpGet:
-              path: /health
+              path: /_ah/health
```
Comment (ndeitch) [inline .k8s/deployment_web.tpl.yml:39]: Why this change? `/_ah` is a app engine config, I think `/health` is ok

File: Dockerfile [hunk 2]
```diff
@@ -1,27 +1,24 @@
-FROM node:14.15.0-alpine3.11 As dev
+FROM node:14.15.0-alpine3.11 AS dev
 
-WORKDIR /usr/src/app
+ARG NODE_AUTH_TOKEN
```
Comment (ndeitch) [inline Dockerfile:3]: Don't you have to pass this ARG on `docker build`?

File: .k8s/deployment_web.tpl.yml (hunk 1 above)
Comment (joaofnds) [reply .k8s/deployment_web.tpl.yml:39]: Should I change the following line then? https://github.com/skore-io/skore-ts-template/blob/8da3b29eec9bc0591eb5c8e89974541fbafdae73/src/main.ts#L48

File: Dockerfile (hunk 2 above)
Comment (joaofnds) [reply Dockerfile:3]: I surely do! Great catch! 👌

File: .k8s/deployment_web.tpl.yml (hunk 1 above)
Comment (ndeitch) [reply .k8s/deployment_web.tpl.yml:39]: IMHO yes

**Review by ndeitch — APPROVED** [review]
